pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
IIRC this is here because in local mode the pipeline automatically copies the output to the location of the GAPIC config specified in the artman config. That means that if you have any hand edits to an existing GAPIC config that you haven't `git commit`d, you will lose them.

@shinfan Can you confirm that?